charts/kubeai/Chart.yaml

```diff
@@ -41,7 +29,7 @@ home: https://www.substratus.ai
 
 maintainers:
 - name: nstogner
-  email: nicholas.stogner@gmail.com
+  url: https://www.linkedin.com/in/nstogner/
```

Comment:
I think we need an email later if we upload to artifact hub, but fine with removing for now